src/varity/vcf_to_hgvs/protein.clj

```diff
@@ -197,12 +195,14 @@
      :c-ter-adjusted-alt-prot-seq (codon/amino-acid-sequence
                                    (cond-> ter-site-adjusted-alt-seq
                                      (= strand :reverse) util-seq/revcomp))
-     :alt-rg (-> rg
-                 (assoc :exon-ranges alt-exon-ranges*)
-                 (update :cds-start apply-offset*)
-                 (update :cds-end apply-offset*)
-                 (update :tx-end apply-offset*))
-     :ref-include-ter-site ref-include-ter-site}))
+     :alt-rg (when alt-exon-ranges*
+               (-> rg
+                   (assoc :exon-ranges alt-exon-ranges*)
+                   (update :cds-start apply-offset*)
+                   (update :cds-end apply-offset*)
+                   (update :tx-end apply-offset*)))
+     :ref-include-ter-site ref-include-ter-site
+     :exon-intron-boundary-variant (nil? alt-exon-ranges*)}))
```

Comment:
`:unpredicable-exon-ranges` semantically?

The `alt-exon-ranges` can become `nil` not only due to alterations on the exon-intron boundary, but also due to other possible conditions.